<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>446</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="descriptorIndexStartsAt0Label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Descriptor Index Starts at 0 (Coming later)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="descriptorIndexStartsAt0CheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="primarilyPreferredLocaleLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Primarily Preferred Locale (Coming loon)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="primarilyPreferredLocaleComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="secondarilyPreferredLocaleLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Secondarily Preferred Locale (Coming loon)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="secondarilyPreferredLocaleComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="automaticPNGDownloadLabel">
        <property name="text">
         <string>Automatic Entity Logo Download</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>16</number>
        </property>
        <item>
         <widget class="QCheckBox" name="automaticPNGDownloadCheckBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="clearLogoCacheButton">
          <property name="text">
           <string>Clear Cache</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="automaticCheckForUpdatesLabel">
        <property name="text">
         <string>Automatic Check For Updates</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="automaticCheckForUpdatesCheckBox"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="checkForBetaVersionsLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Check For Beta Versions</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkForBetaVersionsCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="themeColorLabel">
        <property name="text">
         <string>Accent Color</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="themeColorComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Connection Matrix</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="transposeConnectionMatrixLabel">
        <property name="text">
         <string>Show Talkers On Top</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="transposeConnectionMatrixCheckBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="alwaysShowArrowTipConnectionMatrixLabel">
        <property name="text">
         <string>Always Show Arrow Tip</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="alwaysShowArrowTipConnectionMatrixCheckBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="alwaysShowArrowEndConnectionMatrixLabel">
        <property name="text">
         <string>Always Show Arrow End</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="alwaysShowArrowEndConnectionMatrixCheckBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="showMediaLockedDotLabel">
        <property name="text">
         <string>Show Media Locked State (Dot) In Intersection</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="showMediaLockedDotCheckBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="allowCRFAudioConnectionLabel">
        <property name="text">
         <string>Allow CRF-Audio Connection</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="allowCRFAudioConnectionCheckBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="collapsedByDefaultLabel">
        <property name="text">
         <string>Collapsed by Default</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="collapsedByDefaultCheckBox"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="showEntitySummaryLabel">
        <property name="text">
         <string>Show Entity Summary</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="showEntitySummaryCheckBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>ATDECC Controller</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="enableAEMCacheLabel">
        <property name="text">
         <string>Enable AEM Cache</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="enableAEMCacheCheckBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="enableAdvertisingLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Enable Advertising</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="enableAdvertisingCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="controllerIDLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Controller Sub ID (if possible)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="qtMate::widgets::TextEntry" name="controllerIDLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="fullAEMEnumerationLabel">
        <property name="text">
         <string>Full AEM Enumeration</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="fullAEMEnumerationCheckBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="discoveryDelayLabel">
        <property name="text">
         <string>Discovery Delay (in sec, 0=manual)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qtMate::widgets::TextEntry" name="discoveryDelayLineEdit">
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="enableFastEnumerationLabel">
        <property name="text">
         <string>Enable Fast Enumeration</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="enableFastEnumerationCheckBox">
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Network</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="protocolLabel">
        <property name="text">
         <string>Protocol</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="protocolComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="interfaceTypesLabel">
        <property name="text">
         <string>Interface Types</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListView" name="interfaceTypeList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qtMate::widgets::TextEntry</class>
   <extends>QLineEdit</extends>
   <header>QtMate/widgets/textEntry.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>descriptorIndexStartsAt0CheckBox</tabstop>
  <tabstop>primarilyPreferredLocaleComboBox</tabstop>
  <tabstop>secondarilyPreferredLocaleComboBox</tabstop>
  <tabstop>automaticPNGDownloadCheckBox</tabstop>
  <tabstop>clearLogoCacheButton</tabstop>
  <tabstop>automaticCheckForUpdatesCheckBox</tabstop>
  <tabstop>checkForBetaVersionsCheckBox</tabstop>
  <tabstop>themeColorComboBox</tabstop>
  <tabstop>enableAEMCacheCheckBox</tabstop>
  <tabstop>enableAdvertisingCheckBox</tabstop>
  <tabstop>controllerIDLineEdit</tabstop>
  <tabstop>protocolComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
